core/config/v2/types.go

```diff
@@ -30,7 +31,8 @@ type Core struct {
 
 	TelemetryIngress *TelemetryIngress
 
-	Log *Log
+	AuditLogger audit.AuditLoggerConfig
```

Comment:
Does this have to be a pointer to prevent printing zero values in the toml?
```suggestion
	AuditLogger *audit.AuditLoggerConfig
```

Reply:
Does it? I would prefer this not be a pointer to cut down on the number of places it could accidentally dereferenced.

Reply:
Yes. It must be a pointer to distinguish between omission and empty values when marshaling and unmarshaling.

Reply:
`nil` is meaningful here, and they get filled in with default values before use. This process is tested for completeness as well.

Reply:
Converting to pointer